plugins/logging/pkg/agent/drivers/kubernetes_manager/kubernetes_manager.go

```diff
@@ -321,8 +321,10 @@ func (m *KubernetesManagerDriver) buildEmptyCollector() *opnicorev1beta1.Collect
 			ImageSpec: opnimeta.ImageSpec{
 				ImagePullPolicy: lo.ToPtr(corev1.PullAlways),
 			},
-			SystemNamespace: m.Namespace,
-			AgentEndpoint:   otel.AgentEndpoint(serviceName),
+			SystemNamespace:          m.Namespace,
+			AgentEndpoint:            otel.AgentEndpoint(serviceName),
+			AggregatorOTELConfigSpec: opnicorev1beta1.NewDefaultAggregatorOTELConfigSpec(),
+			NodeOTELConfigSpec:       opnicorev1beta1.NewDefaultNodeOTELConfigSpec(),
```

Comment:
Not sure if this change is also required in 
https://github.com/rancher/opni/blob/189a8eb14de15824addbe41f8f8c28d6f48cc24b/plugins/metrics/pkg/agent/drivers/opni_manager_otel/otel_driver.go#L204-L222

Reply:
Wasn't aware of this part, thanks. Something that is still unclear to me: whenever an empty collector needs to be created by the metrics plugin, it will use the default configuration. 

However, I've added the method below so that whenever `nodeOtelCollectorSpec` is updated, the collector is reconciled with the new config (there's an equivalent for the aggregator too). It should also work for metrics, right?

https://github.com/rancher/opni/blob/51262a3d001cdb0e5e79909cb9f3c9f53f108d7e/pkg/resources/collector/workloads.go#L75

Reply:
yeah